boot/bootutil/src/bootutil_priv.h

```diff
@@ -62,16 +62,24 @@ struct flash_area;
 /** Number of image slots in flash; currently limited to two. */
 #define BOOT_NUM_SLOTS                  2
 
-#if !defined(MCUBOOT_OVERWRITE_ONLY)
+#if defined(MCUBOOT_OVERWRITE_ONLY) && defined(MCUBOOT_SWAP_MOVE)
+#error "Please enable only one of MCUBOOT_OVERWRITE_ONLY or MCUBOOT_SWAP_MOVE"
+#endif
+
+#if !defined(MCUBOOT_OVERWRITE_ONLY) && !defined(MCUBOOT_SWAP_MOVE)
 #define MCUBOOT_USE_SCRATCH 1
 #endif
 
+#define BOOT_STATUS_OP_MOVE     1
```

Comment:
do you expect support more than one swap algorithm in any mcuboot build?

Reply:
No, why?

Reply:
I misunderstood purpose of these defines, as I reviewed their usage later. nevermind.